buildenv/jenkins/openjdk_tests

```diff
@@ -342,6 +342,12 @@ timestamps{
                     }
                 }
 
+            if (PLATFORM == "ppc64_aix" && env.JENKINS_URL.contains("hyc-runtimes") && env.LABEL == "") {
```

Comment:
It seems to work but should it be params.LABEL instead?

Reply:
I think the logic will override the LABEL assignment in the block above, is that the intent?

Reply:
I moved it into the block above.

Reply:
It doesn't override the LABEL, it appends to it.